client/project_test.go

```diff
@@ -72,7 +74,7 @@ var _ = Describe("Project", func() {
 			mockedResponse.Description = payload.Description
 
 			httpCall = mockHttpClient.EXPECT().
-				Put("/projects/" + mockProject.Id, payload, gomock.Any()).
+				Put("/projects/"+mockProject.Id, payload, gomock.Any()).
```

Comment:
why no spaces?
you did it on other files too

Reply:
That's GoLand doing it 🤷 
Guess it's per go specification